apps/ledger-live-mobile/src/components/DeviceJob/steps.tsx

```diff
@@ -59,12 +67,12 @@ export const connectingStep: Step = {
     );
   },
   run: meta =>
-    withDevice(meta.deviceId)(() => from([meta])).pipe(
+    withDevice(meta.deviceId as string)(() => from([meta])).pipe(
```

Comment:
Why not make `withDevice(deviceId: DeviceModelId)` ? And for all other functions called under

Reply:
This is located in `live-common`, the scope of this PR is LLM.

```
export const withDevice =
  (deviceId: string) =>
```